docs/advanced-features/custom-app.md

```diff
@@ -14,15 +14,17 @@ Next.js uses the `App` component to initialize pages. You can override it and co
 To override the default `App`, create the file `./pages/_app.js` as shown below:
 
 ```jsx
+// Only uncomment this import and the method below if you have blocking data requirements for
```

Comment:
Please move this comment back, I don't think this commented out import requires explanation.

Reply:
@Timer I moved there because if someone reads the comment, he will most likely uncomment the method but not the import, and it will initially fail